### ISSUE TITLE
github repository import template

### ISSUE THREAD

**Issue by filename24** [discordjs/discord.js#9812]
### Which application or package is this feature request for?

create-discord-bot

### Feature

Creating a template using a GitHub repository

### Ideal solution or implementation

I think we can provide a function that first checks if the GitHub repository is found, and then downloads and renames the repository.

**Comment by Jiralite** [comment]
This seems like a fundamental Git query accomplished with `git clone git@github.com:filename24/filename24.git renamed`, for example, no?

**Comment by filename24** [comment]
> This seems like a fundamental Git query accomplished with `git clone git@github.com:filename24/filename24.git renamed` for example, no?

Yes, that's right, but if you look at the code, you're doing a git clone using the [REPLACE_ME] name in the package.json.

**Comment by Jiralite** [comment]
I'm struggling to understand now. Isn't the solution to that to follow the respective repository's instructions?

**Comment by filename24** [comment]
If you're going to write a template, you're most likely going to write a git clone. But if you're going to do a create-discord-bot, why not build something that installs the module, automatically removes the cloned .git fullder, and automatically does a git init, for example `create-discord-bot hellobot --example https://github.com`?

**Comment by Jiralite** [comment]
If you mean this should be a tool to install anyone's template, we will not be doing that. _This_ is the template. Follow a repository's instructions to learn how to deal with their templates.
